flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/pom.xml

```diff
@@ -29,7 +29,7 @@ limitations under the License.
     <artifactId>flink-cdc-pipeline-connector-paimon</artifactId>
 
     <properties>
-        <paimon.version>0.9.0</paimon.version>
+        <paimon.version>1.0.0</paimon.version>
```

Comment:
According to [Paimon blog](https://paimon.apache.org/releases/1.0.1), the first officially released stable version of Paimon should be 1.0.1 instead of 1.0.0.